jobs/build-node-image.Jenkinsfile

```diff
@@ -98,7 +100,8 @@ lock(resource: "build-node-image") {
                                                 manifest_tag_staging: container_registry_staging_manifest_tag,
                                                 secret: "id=yumrepos,src=${yumrepos_file}", // notsecret (for secret scanners)
                                                 from: build_from,
-                                                extra_build_args: ["--security-opt label=disable", "--mount-host-ca-certs", "--force"])
+                                                extra_build_args: ["--security-opt label=disable", "--mount-host-ca-certs", "--force", 
+                                                "--label=org.opencontainers.image.version=${container_registry_staging_image_tag}.${timestamp}"])
```

Comment:
Should we also set `io.openshift.build.versions` ?

Reply:
Both of these are important:
```
  "io.openshift.build.version-display-names": "machine-os=Red Hat Enterprise Linux CoreOS",
  "io.openshift.build.versions": "machine-os=419.96.202503251418-0",
```
...as the get consumed by `oc adm release new` to create the `displayVersions."machine-os"` attribute in the resulting payload.

Reply:
☝🏼 is wrong. It is intentional that these two labels are retained from the rhel layer.